docs/DafnyRef/UserGuide.md

```diff
@@ -233,7 +233,7 @@ implementation.
 
 The options relevant to this command are
 - those relevant to the command-line itself
-   - `--warn-as-errors` --- turn all warnings into errors, which alters [dafny's exit code](#sec-exit-codes)
+   - `--allow-warnings` --- return a success [exit code](#sec-exit-codes), even when there are warnings 
```

Comment:
Not true though since `dafny run` or `dafny test` still could return a non-zero exit code.